vital/plugin_loader/plugin_info.h

```diff
@@ -0,0 +1,38 @@
+/*ckwg +29
+ * Copyright 2018 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
+ *    to endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef VITAL_PLUGIN_LOADER_PLUGIN_INFO_H
+#define VITAL_PLUGIN_LOADER_PLUGIN_INFO_H
+
+#define PLUGIN_INFO(NAME, DESCRIP)             \
```

Comment:
I wonder if this should instead expand to `PLUGIN_INFO_WITH_ORG(NAME, DESCRIPTION, KWIVER_DEFAULT_PLUGIN_ORGANIZATION)`? This would be more convenient if some user wants/needs to specify a different organization. 

Note: this implies making the organization also a static member, and also probably:
```c++
#ifndef KWIVER_DEFAULT_PLUGIN_ORGANIZATION
#define KWIVER_DEFAULT_PLUGIN_ORGANIZATION "Unspecified"
#endif
```
...here, rather than in `plugin_registrar.h`.

(Feel free to bikeshed on the name of the macro.)

Reply:
Fixed the organization problem by adding it to the CMake file.